tests/Backend/Teradata/ImportTypedTableTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Keboola\Test\Backend\Teradata;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
+use Keboola\TableBackendUtils\Column\Teradata\TeradataColumn;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+use Keboola\Test\Backend\Workspaces\ParallelWorkspacesTestCase;
+
+class ImportTypedTableTest extends ParallelWorkspacesTestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $token = $this->_client->verifyToken();
+
+        if (!in_array('tables-definition', $token['owner']['features'])) {
+            $this->markTestSkipped(sprintf('Tables definition feature is not enabled for project "%s"', $token['owner']['id']));
+        }
+
+        $this->initEmptyTestBucketsForParallelTests();
+    }
+
+    public function testLoadTypedTablesConversionError(): void
+    {
+        $fullLoadFile = __DIR__ . '/../../_data/users.csv';
+
+        $bucketId = $this->getTestBucketId(self::STAGE_IN);
```

Comment:
```suggestion
        $bucketId = $this->getTestBucketId();
```

Reply:
je to takhle ve vsech testech, tak jsem to tam taky nechal :) Ale jo, muze to jit pryc.